### PR TITLE
[DT-552][risk=no] Bump Tanagra tag to 0.0.580

### PR DESCRIPTION
**Pull request by dolbeew**
- Update Tanagra tag to include export column underlay changes
- Rename auth env var to match change in Tanagra

### CODE REVIEW

File: api/libproject/devstart.rb [hunk 1]
```diff
@@ -2618,7 +2618,7 @@ def deploy_tanagra(cmd_name, args)
   ENV.update({"TANAGRA_ACCESS_CONTROL_BASE_PATH" => env_project.fetch(:api_endpoint_host)})
   ENV.update({"TANAGRA_ACCESS_CONTROL_MODEL" => env_project.fetch(:tanagra_access_control_model)})
   ENV.update({"TANAGRA_AUTH_DISABLE_CHECKS" => env_project.fetch(:tanagra_auth_disable_checks)})
-  ENV.update({"TANAGRA_AUTH_BEARER_TOKEN" => env_project.fetch(:tanagra_auth_bearer_token)})
+  ENV.update({"TANAGRA_AUTH_GCP_ACCESS_TOKEN" => env_project.fetch(:tanagra_auth_bearer_token)})
```
Comment (freemabd) [inline api/libproject/devstart.rb:2621]: Can we change the variables to match?
```suggestion
  ENV.update({"TANAGRA_AUTH_GCP_ACCESS_TOKEN" => env_project.fetch(:tanagra_auth_gcp_access_token)})
```

File: api/libproject/environments.rb [hunk 2]
```diff
@@ -55,7 +55,7 @@ def env_with_defaults(env, config)
     :tanagra_auth_disable_checks => "false",
     :tanagra_auth_bearer_token => "true",
```
Comment (freemabd) [inline api/libproject/environments.rb:56]: Need to change this if you accept the change above.
```suggestion
    : tanagra_auth_gcp_access_token => "true",
```

**Review by freemabd — APPROVED** [review]
Looks good. Left a few suggestions and remember that you need to publish the new index into the test security perimeter. Please put an entry for the publish in [CDR Build Log](https://docs.google.com/document/d/1WZvkrIaaVWFM6KD742DbEwwbK03OC_RtOreCMXFU6xQ/edit#heading=h.lj4xe5tjuveg)

File: api/libproject/devstart.rb (hunk 1 above)
Comment (dolbeew) [reply api/libproject/devstart.rb]: Makes sense, changed

**Pull request merged**